1-Draft/RFCNNNN-Test-Connection.md

```diff
@@ -0,0 +1,206 @@
+---
+RFC: XXXX
+Author: Joel Sallow
+Status: Draft
+Version: 1.0
+Area: Cmdlets
+Comments Due: 6/10/19
+Plan to Implement: Yes
+---
+
+# Proposed Changes to `Test-Connection` Cmdlet
+
+## Syntax
+
+```
+# All Sets
+Test-Connection [-TargetName] <string[]> [-IPv4] [-IPv6] [-ResolveDestination] [-TimeoutSeconds <int>] [-Quiet] [<CommonParameters>]
+
+# Set 1
+[-Ping] [-Count <int>] [-Delay <int>] [-Source <string>] [-MaxHops <int>] [-BufferSize <int>] [-DontFragment]
```

Comment:
This would be the default parameter set, correct?

Reply:
Yep!